pkg/webui/lib/field-description-messages.js

```diff
@@ -0,0 +1,174 @@
+
+  appSKeyDescription:
+    'After activation, this encryption key is used to secure messages which carry a payload.',
+  appSKeyLocation:
+    'For OTAA, the key is generated by the Network Server. If using ABP, you must create one and manually enter it in the end device and The Things Stack.',
+
+  nwkSKeyDescription:
+    'After activation, this encryption key is used to secure messages which do not carry a payload.',
+  nwkSKeyLocation:
+    'For OTAA, it is created by the Network Server. If using ABP, you must create one and manually enter it in the end device and The Things Stack.',
+
+  lwVersionDescription:
+    'The LoRa Alliance LoRaWAN specification your end device conforms to, which defines which Media Access Control features it supports.',
+  lwVersionLocation:
+    'The LoRaWAN version for your end device should be provided by the manufacturer in a datasheet as LoRaWAN version or LoRaWAN specification',
+  lwVersionAbsence:
+    'Contact your manufacturer or reseller, since specifying the wrong version can lead to complex issues when the Network Server provides the end device with unsupported configuration (MAC) commands.',
+
+  regParameterDescription:
+    'The Regional Parameters specify frequency, dwell time, and other communication settings for different geographical areas. The Regional Parameters version is the version of the LoRa Alliance specification which your device supports.',
+  regParameterLocation:
+    'The Regional Parameters version should be provided by the end device manufacturer in a datasheet.',
+  regParameterAbsence:
+    'Contact your manufacturer or reseller, since specifying the wrong version can lead to complex issues when the Network Server provides the end device with unsupported configuration (MAC) commands.',
+
+  classDescription:
+    'The LoRaWAN specification defines three end device types. All LoRaWAN end devices must implement Class A, whereas Class B and Class C are extensions to the specification of Class A devices.',
+  classLocation:
+    'The class capabilities of your end device should be provided by the manufaturer in a datasheet.',
+  classAbsence:
+    'All end devices support Class A, so you can always use it. Contact your manufacturer or reseller if you can not find information about what additional classes your end device supports.',
+
+  rx1DataRateOffsetDescription:
+    'The Data Rate Offset sets the offset between the uplink data rate and the downlink data rate used to communicate with the End Device during the first reception slot (RX1).',
+
+  dataRateIndexDescription:
+    'The Data Rate Index specifies which data rate downlink communications will use, as given in the Regional Parameters.',
+})
+
+const descriptions = {
+  [GLOSSARY_IDS.FREQUENCY_PLAN]: {
+    description: m.freqPlanDescription,
+    location: m.freqPlanLocation,
+  },
+  [GLOSSARY_IDS.DEV_EUI]: {
+    description: m.devEuiDescription,
+    location: m.devEuiLocation,
+    absence: m.devEuiAbsence,
+  },
```

Comment:
What do you think about also adding the field title message to these definitions so we have everything in one place and then making the title prop optional for the `<Field />` component?

Reply:
From what I can see the `title` prop is already optional for `<Form.Field />`.  What is the benefit to set the `title` prop directly for some fields and pre-define that in `field-description-messages` for the others?

Reply:
Yeah, it was just something that came to my mind but would likely generate too much abstraction. I liked the idea of sort of having "well known" fields of which we could store all information in one place (e.g. also field type and validation logic) and then use throughout. We currently use some fields in multiple forms.